protobuf/lib/src/protobuf/utils.dart

```diff
@@ -35,4 +35,29 @@ bool _areByteDataEqual(ByteData lhs, ByteData rhs) {
   return _areListsEqual(asBytes(lhs), asBytes(rhs));
 }
 
+List<T> sorted<T>(Iterable<T> list) => new List.from(list)..sort();
+
+// Jenkins hash functions
+
+int _combine(int hash, int value) {
```

Comment:
Either give these a more distinct name or collect them as static methods in a (private) class